relayer-cli/src/commands/query/client.rs

```diff
@@ -19,19 +19,25 @@ use crate::conclude::Output;
 use crate::error::{Error, Kind};
 use crate::prelude::*;
 
+// TODO: refactor commands: why is chain_id an `Option`? simpler to give it `ChainId` type.
+//  see the tx/client.rs or tx/packet.rs for simpler approaches.
```

Comment:
You mean `client_id`? Yes, this file slipped through the cracks. We had everything `String` initially and then moved gradually to typed options for most CLIs.

Reply:
I meant the `chain_id` field in the command:


https://github.com/informalsystems/ibc-rs/blob/9d0a7b8f11a252e68e039a976910edd33107f074/relayer-cli/src/commands/query/client.rs#L24-L26

No reason to wrap it in an Option there. But also applies to the `client_id` which should not be an option either and should be type `ClientId`.

Reply:
changed in ecb53189a66496c714ebec00f1769132776e191f. For the rest I opened https://github.com/informalsystems/ibc-rs/issues/572.